scripts/download-storybook-data.ts

```diff
@@ -29,6 +29,19 @@ const LSD_DOCS_DIR = path.join(DOCS_DIR, './lsd')
 const COMPONENTS_DIR = path.join(LSD_DOCS_DIR, './components')
 const DESIGN_TOKENS_DIR = path.join(LSD_DOCS_DIR, './design-tokens')
 
+const getGlobalControls = (story): GlobalControls[] => {
+  if(story.name === 'Colors') {
+    return ["themeColor"]
+  } else if(story.name === 'Typography') {
+    return ["themeFont"]
+  } else if(story.name === 'Spacing') {
+    // On the spacing page, there's no need for a themeColor or themeFont dropdown
+    return []
+  }
+
+  return ["themeColor", "themeFont"]
+}
+
```

Comment:
Added this function to the `download-storybook-data` file. Was this what you were thinking @jeangovil ?